error: duplicate #[pin] attribute
 --> tests/ui/pin_project/add-attr-to-struct.rs:9:5
  |
9 |     #[pin] //~ ERROR duplicate #[pin] attribute
  |     ^^^^^^

error: #[pin] attribute may only be used on fields of structs or variants
  --> tests/ui/pin_project/add-attr-to-struct.rs:13:1
   |
13 | #[add_pin_attr(struct)] //~ ERROR #[pin] attribute may only be used on fields of structs or variants
   | ^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `add_pin_attr` (in Nightly builds, run with -Z macro-backtrace for more info)
